sui/its.js

```diff
@@ -5,13 +5,18 @@ const { addBaseOptions, addOptionsToCommands, getWallet, printWalletInfo, broadc
 const { bcs } = require('@mysten/sui/bcs');
 
 const SPECIAL_CHAINS_TAGS = {
-    ALL_EVM: 'all-evm', // All EVM chains that have InterchainTokenService deployed
+    ALL: 'all',
 };
 
+function getAllTrustedChains(config) {
+    const allChains = Object.keys(config.chains);
+    return allChains;
+}
+
 function parseTrustedChains(config, trustedChains) {
-    if (trustedChains === SPECIAL_CHAINS_TAGS.ALL_EVM) {
-        const evmChains = Object.keys(config.chains).filter((chain) => config.chains[chain].contracts?.InterchainTokenService?.address);
-        return evmChains;
+    if (trustedChains[0] === SPECIAL_CHAINS_TAGS.ALL) {
+        const allChains = getAllTrustedChains(config);
+        return allChains.filter((chain) => config.chains[chain].contracts?.InterchainTokenService?.address);
     }
 
     return trustedChains;
```

Comment:
If a special tag isn't used, should at least check that the chain names correspond to correct `axelarId` in the config and warn if not (it's possible a third party chain, e..g xrpl isn't in the config but want to catch user errors in other scenarios)